SPECS/clamav/clamav.spec

```diff
@@ -126,6 +127,7 @@ fi
 %{_libdir}/*.so.*
 %{_libdir}/pkgconfig/*.pc
 %{_libdir}/systemd/system/*
+%{_libdir}/*.a
```

Comment:
This is likely not needed and can be excluded (especially in the core package)

Reply:
There is a reason why I added this file:
![image](https://github.com/microsoft/azurelinux/assets/38226164/08a44658-86ac-40c4-b901-ee45c535d719)
So, would you like me to add the file in a less wide way?
%{_libdir}/*.a -> %{_libdir}/*_rust.a or smth like %{_libdir}/lib%{name}*.a
Or make this file be removed somewhere earlier?

Reply:
I currently changed it to  %{_libdir}/lib%{name}*.a, let me know what you think

Reply:
Removed it after upgrading to 1.0.6

Reply:
I was suggesting to simply delete the file and not package it. But it looks like this is not necessary for the 1.0.6 version